atd-vzd/migrations/default/1699990226463_alter_table_public_atd_txdot_person_add_column_peh_fl/up.sql

```diff
@@ -0,0 +1,4 @@
+alter table "public"."atd_txdot_person" add column "peh_fl" boolean
+ null;
+
+COMMENT ON COLUMN public.atd_txdot_person.peh_fl IS 'Boolean flag that indicates whether the person was experiencing homelessness at the time of the crash.';
```

Comment:
i think using postgres comments as much as we can especially for columns is great practice!! ive been noticing we have been taking advantage of this more as a team lately

Reply:
I need to do much better about this - thank you for mentioning it. I agree with you, it's nothing but good. 🚀